.travis.yml

```diff
@@ -8,8 +8,7 @@ cache:
     - node_modules
 
 script:
-  - npm run build
-  - npm run test
+  - npm run travis
```

Comment:
What's the benefit of collapsing all the steps into one like this? Travis is failing and I'm not really in a position to dig into why, but I think most of the tooling would give a more descriptive error about which step failed if they were all listed separately here.

Reply:
So that I can be sure in advance of pushing code that I've done my due dilligence and travis won't fail.  Now, in this case I made a mistake in my "clean" script and only didn't notice it because the folder I was cleaning still existed.  In the travis run, the folder wasn't there because it was already clean.  I've got a fix for that locally, and I'll push that as soon as I've ensured all the files that need to be linted will be linted (and linted only once).

As for tooling giving descriptive errors, from what i can tell the errors are all laid out because it's just an "&&" of a bunch of separate commands that are run in order.

Reply:
I'm gonna leave this for now, but I'm happy to discuss more when you're back.  In the meanwhile, I'm going to merge to unblock the artists.